setup.cfg

```diff
@@ -7,6 +7,6 @@ with-doctest=1
 doctest-extension=rst
 
 [build_sphinx]
-source-dir = docs/
-build-dir  = docs/_build
+source-dir = doc/
```

Comment:
Yes, this was always broken as it is the setup to build the docs from setuptools, and setup tools launches the build from where `setup.py` is called. So this never worked, you can even see that it has the old `docs` path, because it was never used, and thus never took care of updating.
But now that we can call sphinx-build from any place I thought to make use of this. Because if setuptools can build the docs it can also directly upload them to pythonhosted.org when uploading the package to PyPI. Of course we stopped hosting the docs in pythonhosted long ago in favor of readthedocs.
In circle.yml all 3 statements in the test do the same, is only to test the build from different places a calling from different commands.